src/main/java/core/basesyntax/service/StorageAccess.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service;
+
+import java.util.List;
+import java.util.Map;
+
+public interface StorageAccess {
+    List<String> readData(String fileName);
+
+    void writeData(Map<String, Integer> data, String fileName);
+}
```

Comment:
You must create 5 separeted services 
-read from file
-data parser service
-calculator
-report maker
-write to file

Reply:
All the services must be in package "service", and their impl. classes in service.impl